src/Features/CSharp/Portable/Completion/CompletionProviders/NamedParameterCompletionProvider.cs

```diff
@@ -271,6 +262,15 @@ private static ISet<string> GetExistingNamedParameters(BaseArgumentListSyntax ar
         return null;
     }
 
+    private static IEnumerable<ImmutableArray<IParameterSymbol>>? GetWithElementParameterLists(
```

Comment:
>IEnumerable<ImmutableArray<IParameterSymbol>>? [](http://example.com/codeflow?start=19&length=46)

why nullable?

Reply:
just matching the sibling methods.  you're right that it's not necessary here.